src/Bedrock.Framework.Experimental/Protocols/Kafka/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "2.4"
+
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:4.1.0
+    hostname: zookeeper
+    ports:
+      - 2181:2181
+    environment:
+      - "ZOOKEEPER_CLIENT_PORT=2181"
+      - "KAFKA_JMX_PORT=39999"
+      - "KAFKA_JMX_HOSTNAME=zookeeper"
+    restart: on-failure
+
+  kafka:
+    image: confluentinc/cp-kafka:4.1.0
+    hostname: kafka
+    environment:
+      - "KAFKA_ZOOKEEPER_CONNECT=zookeeper:2181/jmx"
```

Comment:
trivia...
it's more "normal" to specify like
```
    environment:
      KAFKA_BROKER_ID: 1
      ...etc...
```
doesn't make much difference but more inline with most examples out there esp. the confluent ones